spaghetto/src/Spago/Cmd.js

```diff
@@ -1,48 +0,0 @@
-import { execa, execaCommand } from "execa";
-
-const mkOptions = (opts) => {
-  let options = { preferLocal: true, shell: opts.shell };
```

Comment:
Here we always set the `preferLocal: true` option, we should port that to the new code as well

Reply:
`preferLocal: true` [is already the default](https://github.com/JordanMartinez/purescript-node-execa/blob/master/src/Node/Library/Execa.purs#L182)

Reply:
See https://github.com/JordanMartinez/purescript-node-execa/blob/master/src/Node/Library/Execa.purs#L89-L93 for why I use `Just { Nothing, Nothing }`. When unspecified, `cwd` and `execPath` default to `cwd` and the current process' `execPath`.

Reply:
Ah, great then!